qt/python/mantidqt/widgets/codeeditor/tab_widget/codeeditor_tab_presenter.py

```diff
@@ -0,0 +1,42 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+from __future__ import absolute_import
+
+import os
+import subprocess
+import sys
+
+from mantid.kernel import logger
+
+
+class ShowInExplorer(object):
+    @staticmethod
+    def open_directory(path):
+        if sys.platform == "win32":
+            os.startfile(os.path.dirname(path))
+        elif sys.platform == 'darwin':
+            subprocess.check_call(['open', '--', path])
```

Comment:
This needs to be verified that it doesn't error on Mac

Reply:
Confirmed that this does work on macOS. :tada: